contracts/farms/common/RevoFarmBot.sol

```diff
@@ -43,32 +56,27 @@ contract UbeswapFarmBot is ERC20, AccessControl, Pausable {
         bytes32 role
     );
 
+    modifier ensure(uint256 deadline) {
+        require(deadline >= block.timestamp, "FarmBot: EXPIRED");
+        _;
+    }
+
     bytes32 public constant COMPOUNDER_ROLE = keccak256("COMPOUNDER_ROLE");
```

Comment:
why have this role on a contract with no compound method?

Reply:
same with the Compound event

Reply:
maybe Revo events should be moved to a library

Reply:
See my other comment on this